common/src/storage/complicated/counter.rs

```diff
@@ -44,6 +44,14 @@ pub trait Counter {
 
     /// Resets stored value by setting default/starting value.
     fn reset();
+
+    /// Increases and returns stored value.
+    ///
+    /// Should be safe from overflow.
+    fn inc_get() -> Self::Value {
```

Comment:
Why not let `increase()` return the new counter value?

Reply:
it is used only in one place so looks like the method is unnecessary

Reply:
+1